x-pack/plugins/apm/public/components/app/service_overview/service_overview_transactions_table/index.tsx

```diff
@@ -133,22 +156,27 @@ export function ServiceOverviewTransactionsTable({ serviceName }: Props) {
               numBuckets: 20,
               transactionType,
               latencyAggregationType,
-              transactionNames,
+              transactionNames: JSON.stringify(
+                transactionGroups.map(({ name }) => name).sort()
+              ),
+              comparisonStart,
+              comparisonEnd,
             },
           },
         });
       }
     },
-    // only fetches statistics when requestId changes or transaction names changes
+    // only fetches comparison statistics when requestId is invalidated by primary statistics api call
     // eslint-disable-next-line react-hooks/exhaustive-deps
-    [requestId, transactionNames],
+    [requestId],
```

Comment:
maybe leave a comment here mentioning #93942 as a known issue?

Reply:
hmm Do you think it's necessary? To fix this bug nothing has to be changed on this file.